plot-stdin

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
Add example usage at least as a comment here. Even better add it to `README.md`. Even better move `plot-stdin{,.py}` to `plot-stdin/` and add it to `plot-stdin/README.md`